tests/conftest.py

```diff
@@ -1376,7 +1376,6 @@ def minimal_config(request, scout_config):
     config.pop("rank_model_version")
     config.pop("rank_score_threshold")
     config.pop("sv_rank_model_version")
-    config.pop("human_genome_build")
```

Comment:
I find this uneasy at this point, though in principle it would have been better from the start. But we have strict value checking now, and the default hg38 is right for most.